source/jormungandr/jormungandr/instance.py

```diff
@@ -94,8 +95,12 @@ def __init__(self,
                                                 reset_timeout=app.config['CIRCUIT_BREAKER_INSTANCE_TIMEOUT_S'])
         self.georef = georef.Kraken(self)
         self.planner = planner.Kraken(self)
-        self.street_network_service = street_network.StreetNetwork.get_street_network(self,
-                                                                                      street_network_configuration)
+
+        if street_network_configurations is None:
+            street_network_configurations = []
+        self._set_default_street_network_config(street_network_configurations)
```

Comment:
1. Ok for the free function
2. I cannot put L99 and L100 into the free function. because `_set_default_street_network_config` will take the reference of `street_network_configurations` and when it's `None`(instead of a real object), it's not modifiable